commons/monitor/application/application_record_counts.avsc

```diff
@@ -5,8 +5,8 @@
   "doc": "Number of records cached or created.",
   "fields": [
     { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
-    { "name": "recordsCached", "type": ["null", "int"], "doc": "Number of records currently being cached.", "default": null },
-    { "name": "recordsSent", "type": "int", "doc": "Number of records sent since application start." },
+    { "name": "recordsCached", "type": ["null", "long"], "doc": "Number of records currently being cached.", "default": null },
+    { "name": "recordsSent", "type": "long", "doc": "Number of records sent since application start." },
```

Comment:
should we add long instead of replacing int ?

Reply:
IIRC, there was an issue earlier with AvroConnect if more than two union members were used. I don't recall the specifics though...

Reply:
Will the recordsCached be backward compatible ?

Reply:
In JSON encoding: yes, ints will be correctly be parsed as longs.
In binary encoding: yes, ints will be correctly parsed as longs, because of their encoding format (variable length zig-zag encoding).